### PR TITLE
Add ability to hide testnets blockchains

### PR DESCRIPTION
**Pull request by Riddlerrr**
Tasks: https://www.pivotaltracker.com/story/show/175469614 and https://www.pivotaltracker.com/story/show/175483264

I've decided to always show the full list of blockchains for `comakery_admin`'s. Not sure if it correct.:
https://github.com/CoMakery/comakery-server/blob/4304400255de7eda9a2fc40cd463596e8e6a6f68/app/models/wallet.rb#L32

Spotify Premium 2020-10-30 15-27-49

Spotify Premium 2020-10-30 15-13-28

### CODE REVIEW

**Comment by Alexey1100** [comment]
🤔
I think the list of the blockchains for Token and Wallet should be definitely the same, regardless account status, because a wallet using a testnet doesn't have a purpose without a token with the same network.

Which means we can extract all the blockchain availability logic into the Blockchain class.

File: app/models/concerns/belongs_to_blockchain.rb [hunk 1]
```diff
@@ -17,6 +17,14 @@ def self.blockchain_for(name)
       "Blockchain::#{name.camelize}".constantize.new
     end
 
+    def self.testnets_available?
```
Comment (Alexey1100) [inline app/models/concerns/belongs_to_blockchain.rb:20]: Looks like duplicate of

https://github.com/CoMakery/comakery-server/blob/d79dfc56ba84f82f971f5743c313e51f010d0f0d/app/models/blockchain.rb#L27

Comment (Riddlerrr) [reply app/models/concerns/belongs_to_blockchain.rb:20]: Right 👍 , forgot to delete it

File: app/models/wallet.rb [hunk 2]
```diff
@@ -16,12 +16,17 @@ class Wallet < ApplicationRecord
...
+    available_blockchains.map(&:key) - account.wallets.pluck(:_blockchain)
   end
 
   private
 
     def ore_id_and_pending?
       pending? && ore_id?
     end
+
+    def testnet_blockchains_available?
```
Comment (aquabu) [inline app/models/wallet.rb:29]: remove?

File: spec/models/blockchain_spec.rb [hunk 3]
```diff
@@ -7,6 +7,24 @@
...
+      is_expected.to include Blockchain::BitcoinTest
+      is_expected.to include Blockchain::Bitcoin
+    end
+
+    it 'do not returns testnets if TESTNETS_AVAILABLE set to false' do
+      ENV['TESTNETS_AVAILABLE'] = 'false'
+
+      is_expected.not_to include Blockchain::BitcoinTest
+      is_expected.to include Blockchain::Bitcoin
+    end
```
Comment (aquabu) [inline spec/models/blockchain_spec.rb:26]: may need to reset the ENV variable to what it started as after each test

probably should write a test to confirm null testnet value has the same result as true value

**Review by aquabu — APPROVED** [review]

**Pull request merged**